pkg/mocai/registration.go

```
@@ -0,0 +1,63 @@
+	return registrationID, nil
+}
+
+// generates a random registration number in Portuguese.
+func GenerateID(locale string, length int) (string, error) {
+	switch locale {
+	case "pt-br":
+		if length < 2 {
+			panic("Length must be at least 2 to include the checksum")
+		}
+
+		rand.Seed(time.Now().UnixNano())
```

golangci-lint (1.62.2)

SA1019: rand.Seed has been deprecated since Go 1.20 and an alternative has been available since Go 1.0: As of Go 1.20 there is no reason to call Seed with a random value. Programs that call Seed with a known value to get a specific sequence of results should use New(NewSource(seed)) to obtain a local random generator. 

(staticcheck)